scripts/windows/specs/PY34-win32.txt

```diff
@@ -0,0 +1,24 @@
+# List packages that are/would be required for installation
+
+--find-links=https://orange.biolab.si/download/files/wheelhouse
+--only-binary numpy,scipy,Bottleneck,scikit-learn,PyQt5
+
+AnyQt==0.0.8
+numpy==1.12.1
+scipy==0.16.1
```

Comment:
If this is still relevant, could you also update scipy to 0.19 because of #2367?